addons/html_builder/static/tests/builder_action.test.js

```diff
@@ -10,3 +11,20 @@ test("trigger mobile view", async () => {
     await contains("button[data-action='mobile']").click();
     expect(".o_website_preview.o_is_mobile").toHaveCount(1);
 });
+
+test("top window url", async () => {
+    let websiteBuilder;
+    patchWithCleanup(WebsiteBuilder.prototype, {
+        setup() {
+            websiteBuilder = this;
+            this.props.action.context = {
+                params: {
+                    path: "/web/content/",
+                },
+            };
+            super.setup();
+        },
+    });
+    await setupWebsiteBuilder(`<h1> Homepage </h1>`);
+    expect(websiteBuilder.initialUrl).toBe("/website/force/1?path=%2F");
```

Comment:
I made a test here but I do not really like it: it does not bring more than the code itself. To be discussed

Reply:
We can discuss in vocal